java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -3023,4 +3024,69 @@ public void touch(BaseClient client) {
         assertEquals(2, client.touch(new String[] {key1, key2}).get());
         assertEquals(0, client.touch(new String[] {key3}).get());
     }
+
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void bitcount(BaseClient client) {
```

Comment:
Combine all in one test method

Reply:
Thanks! Addressed in https://github.com/Bit-Quill/glide-for-redis/pull/263/commits/2575b0125be5640b722ee685c33c9fd624ae1b00